core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceInputCheckerServiceSpec.js

```diff
@@ -0,0 +1,89 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for MultipleChoiceInputCheckerService.
+ */
+
+describe('Multiple choice input checker service', function() {
+  beforeEach(module('oppia'));
+
+  var AnswerGroupObjectFactory, mcics, customizationArgs,
```

Comment:
Define each variable on a separate line, and end each line with a semicolon. Also, define customizationArgs directly rather than declaring it here and then redefining it below.